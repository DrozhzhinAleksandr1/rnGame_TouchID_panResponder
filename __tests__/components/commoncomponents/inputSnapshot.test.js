import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { Input } from '../../../app/components/commoncomponents/input';

it('create snapshot Input', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<Input />);
    expect(toolbarSnapshot).toMatchSnapshot();
});