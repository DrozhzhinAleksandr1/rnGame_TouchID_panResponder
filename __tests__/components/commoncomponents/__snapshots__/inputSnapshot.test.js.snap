// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot Input 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "width": "100%",
    }
  }
>
  <Component
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderColor": "#ffffff",
          "borderWidth": 2,
          "flexDirection": "row",
          "height": 40,
          "justifyContent": "space-between",
          "paddingLeft": 15,
          "paddingRight": 15,
          "width": 296,
        },
        null,
      ]
    }
  >
    <TextInput
      allowFontScaling={true}
      onBlur={[Function]}
      onChangeText={[Function]}
      placeholderTextColor="#CECECE"
      selectionColor="#5A6772"
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "color": "#000",
            "fontFamily": "Roboto-Light",
            "fontSize": 18,
            "height": "100%",
            "padding": 0,
            "width": 252,
          },
          null,
        ]
      }
      underlineColorAndroid="white"
      value=""
    />
  </Component>
  <ShowWarning
    isshowwarning={false}
  />
</Component>
`;
